src/herder/HerderImpl.cpp

```diff
@@ -893,6 +893,20 @@ HerderImpl::getJsonQuorumInfo(NodeID const& id, bool summary, bool fullKeys,
     Json::Value ret;
     ret["node"] = mApp.getConfig().toStrKey(id);
     ret["slots"] = getSCP().getJsonQuorumInfo(id, summary, fullKeys, index);
+    Json::Value transitive;
+    using CS = QuorumMapIntersectionState::CheckingState;
+    switch (mLastQuorumMapIntersectionState.mCheckingState) {
```

Comment:
Assuming that quorum checking takes 5 seconds and for each new ledger we have new quorum map, won't it always display -1 here, as it would be for almost all of the time in the RUNNING_CHECK state?

Should we display previous value in that case?

Reply:
I agree with @vogel : it's better to use the last result. I can see on startup using -1 as previous value.
When the value is unknown (-1), given that it's json, it's probably better to emit `null` as a value for that field.

Reply:
Ok, I redid this a bit. New version:
  - has a bool field for `intersection`; we can add a different field for "distance to non-intersection" in the future but I wanted to keep this field easy to interpret. `false` means bad.
  - has fields for the last-checked ledger number and size of transitive quorum checked
  - when `intersection` is false, has a field for when the last-good ledger was, and a potential split as discovered by the checker.

Docs updated accordingly.